.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -55,7 +55,7 @@ Push access to the upstream repository is required in order to publish the new t
   - [ ] `tar -czf target/afterburn-${RELEASE_VER}-vendor.tar.gz vendor`
 
 - publish this release on GitHub:
-  - [ ] open a web browser and [create a GitHub Release](https://github.com/coreos/afterburn/releases/new) for the tag above
+  - [ ] find the new tag in the [GitHub tag list](https://github.com/coreos/afterburn/tags) and click the `triple dots` menu, and create a release for it 
```

Comment:
"triple dots" shouldn't be a `string literal` because that text doesn't appear on the web page.